dist/index.js

```diff
@@ -1,227 +1,6 @@
 /******/ (() => { // webpackBootstrap
```

Comment:
is it typical for GitHub Actions to include dist in the repo? for libraries dist is usually generated pre-publish and never committed

Reply:
To the best of my knowledge - yes.

The `action.yml` specifies the entrypoint. This could be Docker or Node, but no (direct) TypeScript. So therefore, we need to compile the `*.ts` files first, and then make `dist/index.js` available so the Actions Runner can pick it up.

(Thanks for asking, I learned something looking this up!)